stroeer/core/v1/article.proto

```diff
@@ -8,133 +8,318 @@ option go_package = "github.com/stroeer/tapir/go/stroeer/core/v1;core";
   Body body = 7;
+
+  // Technical metadata of the article (optional).
   Metadata metadata = 8;
+
+  // Elements of the article to render as a teaser e.g. on overview pages (optional).
   repeated Element elements = 9;
+
+  // Entities of the article like persons, places etc. (optional).
   repeated string keywords = 10;
+
+  // Unique IDs of articles related to this article (optional).
+  //
+  // Related articles are defined manually in the content management system
+  // by the editorial department.
   repeated int64 onwards = 12;
 }
 
+// Technical metadata of the article.
 message Metadata {
+
+  // State of the article in the content management system (optional).
+  // see https://github.com/stroeer/tapir/issues/64
   string state = 1;
-  //  editorial date: Gültig von
+
+  // Manually set editorial date ('Gültig von') at which the article is valid to deliver on
+  // digital platforms in seconds of UTC time since Unix epoch (optional).
   google.protobuf.Timestamp valid_from = 2;
-  //  editorial date: Gültig bis
+
+  // Manually set editorial date ('Gültig bis') till the article is valid to deliver on
+  // digital platforms in seconds of UTC time since Unix epoch (optional).
   google.protobuf.Timestamp valid_to = 3;
-  //  editorial date: Zuletzt bearbeitet
+
+  // editorial date: Zuletzt bearbeitet
+  // TODO: this seems unused....remove?
   google.protobuf.Timestamp last_edited = 4;
+
   //  editorial date: Zuletzt publiziert
+  // TODO: this seems unused....remove?
   google.protobuf.Timestamp last_published = 5;
+
   // editorial date: Erstellungsdatum
+  // TODO: this seems unused....remove?
   google.protobuf.Timestamp created = 6;
-  // editorial date: Aktualisierungsdatum
+
+  // Manually set editorial date ('Aktualisierungsdatum') at which the article
+  // was updated in seconds of UTC time since Unix epoch (optional).
   google.protobuf.Timestamp updated = 7;
 }
 
+// Elements of the article to render as a teaser e.g. on overview pages.
+//
+// Different types of elements like images or videos share
+// the same message structure distinguished by the ElementType field.
 message Element {
+  // Type of the element.
   ElementType type = 1;
+
+  // TODO: this might be obsolete with https://github.com/orgs/stroeer/teams/api/discussions/1
   repeated ElementRelation relations = 2;
+
+  // Assets of the element like images or videos (optional).
   repeated Asset assets = 3;
+
+  // Children of the element (optional).
   repeated Element children = 4;
 }
 
+// Asset of an element.
 message Asset {
+  // Type of the asset.
   AssetType type = 1;
+
+  // TODO: do we still need the index?
   int32 index = 2;
+
+  // Generic map containing general content and configuration information of
+  // the asset (optional).
+  //
+  // Depending on the type of the asset, this map will contain different
+  // entry sets, e.g. 'source', 'width' and 'height' for images.
+  //
+  // Clients must be resilient to unknown or missing entry sets.
```

Comment:
we should create a gallery of samples for the most important elements as a reference.

Reply:
we should we put those examples? directly in the comment?

Reply:
Or generate html / markdown docs. 😉

Reply:
`for i in 1,2,3 ; do grpcurl $API > docs/$i.buff; done`